app/config/config.php

```diff
@@ -5,6 +5,7 @@
 if (getenv('DATABASE_URL') && $db = parse_url(getenv('DATABASE_URL'))) {
     $db = array_map('rawurldecode', $db);
     $db['path'] = substr($db['path'], 1);
+    if (!isset($db['pass'])) $db['pass'] = '';
```

Comment:
```suggestion
    if (!isset($db['pass'])) {
        $db['pass'] = '';
    }
```

In PSR-2 you always need brackets for conditional statements.

Reply:
Sure, but but then please note this in the project description somewhere.